javascript/introduction-to-js-1/first-splash/number-guessing-game.html

```diff
@@ -61,7 +61,7 @@ <h1>Number guessing game</h1>
           guesses.textContent = 'Previous guesses: ';
         }
 
-        guesses.textContent += userGuess + ' ';
+        guesses.textContent = `${guesses.textContent} ${userGuess}`;
```

Comment:
While the "+=" usage is not wrong here, it's good to update this line so that it matches the template literal assignment we have in [the code in content](https://github.com/mdn/content/blob/0d66181876fd29da72c99140f5828be05a1c9011/files/en-us/learn/javascript/first_steps/a_first_splash/index.md?plain=1#L295) via https://github.com/mdn/content/pull/28603